internal/tiltfile/starkit/fixture.go

```diff
@@ -39,6 +41,7 @@ func (f *Fixture) OnStart(e *Environment) error {
 
 	e.SetPrint(func(t *starlark.Thread, msg string) {
 		_, _ = fmt.Fprintf(f.out, "%s\n", msg)
+		_, _ = fmt.Printf("%s\n", msg)
```

Comment:
?

i'm not sure that all tests would want this?

Reply:
hm, I figured:
1. if the test passes, test runners will hide the output anyway
2. if the test fails, then the output is generally nice to have around
3. tests with Tiltfiles that generate an unmanageable amount of output are likely to be rare

I added this in reaction to a couple of failing tests where the information I needed to debug them was in the tiltfile output. Is there a cost I'm missing that offsets that benefit?

Reply:
the upper tests are a good example of where this logic leads you - you end up with lots of unrelated output that obscures what failed.

Reply:
i'm pretty sure we already have Tiltfiles that already print a ton, so (3) is already not true.

Reply:
> the upper tests are a good example of where this logic leads you - you end up with lots of unrelated output that obscures what failed.

Interesting. I feel like the upper tests are a good example of this logic paying off! In my experience, the relevant failure info is almost always at the end of the test output, where I'd expect it, and then more information is available above if I want it. (aside from that period where it was dumping a stack trace in a defer so it got printed after the failure). I guess we have different tastes.

> i'm pretty sure we already have Tiltfiles that already print a ton

Hmm, I was thinking unmanageable as in "kick the start of the test out of your scrollback buffer". and I haven't seen or conceived of any need to have tests that generate overbearing logs. Maybe I haven't been working on the fun parts of the tiltfile.

Anyway, this is unrelated to this purpose of this PR, so I'll take it out and consider revisiting it if I continue to find it annoying.